app/src/main/java/com/github/alfabravo2013/readyforexams/presentation/create/CreateFragment.kt

```diff
@@ -13,12 +11,9 @@ import com.github.alfabravo2013.readyforexams.presentation.BaseFragment
 import com.github.alfabravo2013.readyforexams.presentation.create.CreateViewModel.OnEvent
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
-class CreateFragment : BaseFragment() {
+class CreateFragment : BaseFragment<FragmentCreateBinding>(FragmentCreateBinding::inflate) {
```

Comment:
And then override onCreateViewBinding and inflate the new layout as FragmentCreateBinding.inflate(inflater, container, false)  . As I said it's kind of similar to what you did, but I feel it's cleaner.